src/kic/datamodel/models/core.py

```diff
@@ -183,3 +229,22 @@ class Meta:
         verbose_name = "object-verzoek"
         verbose_name_plural = "object-verzoeken"
         unique_together = ("verzoek", "object")
+
+
+class VerzoekProduct(APIMixin, models.Model):
+    uuid = models.UUIDField(
+        unique=True, default=uuid.uuid4, help_text="Unieke resource identifier (UUID4)"
+    )
+    verzoek = models.ForeignKey(
+        "datamodel.Verzoek",
+        on_delete=models.CASCADE,
+        help_text="URL-referentie naar het VERZOEK.",
+    )
+    product = models.URLField(
+        null=True,
```

Comment:
nullable text-based fields are usually not a good idea, `blank=True` by itself should be sufficient, no?

Reply:
I think we had this discussion before but I don't agree with the term "text-based fields" as an indicator if it can be nullable or not.

I think a ChoiceField (string-based) that is not required, should be `null` if not filled (because "" is not a valid value). Same for URLField (string-based), where "" is not a valid URL.

For text fields that are meant to store (arbitrary) texts I agree that they should not be nullable.

Anyway, we can adhere to what we used elsewhere for now.